src/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -692,5 +692,153 @@ public static void SyndicationFeed_Rss_WrongDateFormat()
+                XmlDiff diff = new XmlDiff();
+                Assert.True(diff.Compare(filePath, serializeFilePath));
+            }
+            finally
+            {
+                File.Delete(serializeFilePath);
+            }
+        }
+
+        [Fact]
+        [ActiveIssue(24572)]
+        public static void AtomEntryPositiveTest_write()
+        {
+            SyndicationItem item1 = new SyndicationItem("SyndicationFeed released for .net Core", "A lot of text describing the release of .net core feature", new Uri("http://Contoso.com/news/path"));
+            string serializeFilePath = Path.GetTempFileName();
+
+            try
+            {
+                XmlWriter writer = XmlWriter.Create(serializeFilePath, new XmlWriterSettings() { Async = true });
+
+                Atom10ItemFormatter f = new Atom10ItemFormatter(item1);
+                Task task = f.WriteToAsync(writer);
+                Task.WhenAll(task);
+                writer.Close();
+
+                //TODO:
+                // Check file content
+            }
+            finally
+            {
+                File.Delete(serializeFilePath);
+            }
+        }
+
+        [Fact]
+        public static void AtomFeedPositiveTest()
+        {
+            string filePath = @"absolute_rel.xml";
+            string serializeFilePath = Path.GetTempFileName();
+
+            try
+            {
+                XmlReader reader = XmlReader.Create(filePath, new XmlReaderSettings() { Async = true });
+                CancellationToken ct = new CancellationToken();
+                Task<SyndicationFeed> task = SyndicationFeed.LoadAsync(reader, ct);
+                Task.WhenAll(task);
+                SyndicationFeed feedObjct = task.Result;
+
+                XmlWriter writer = XmlWriter.Create(serializeFilePath);
+                Atom10FeedFormatter f = new Atom10FeedFormatter(feedObjct);
+                f.WriteToAsync(writer, ct).GetAwaiter().GetResult();
+                writer.Close();
+
+                CompareHelper ch = new CompareHelper();
+                ch.Diff = new XmlDiff()
+                {
+                    Option = XmlDiffOption.IgnoreComments | XmlDiffOption.IgnorePrefix | XmlDiffOption.IgnoreWhitespace | XmlDiffOption.IgnoreChildOrder | XmlDiffOption.IgnoreAttributeOrder
+                };
+                ch.AllowableDifferences = GetAtomFeedPositiveTestAllowableDifferences();
+                Assert.True(ch.Compare(filePath, serializeFilePath));
+            }
+            finally
+            {
+                File.Delete(serializeFilePath);
+            }
+        }
+
+
+        public static List<AllowableDifference> GetAtomFeedPositiveTestAllowableDifferences()
+        {
+            return new List<AllowableDifference>(new AllowableDifference[]
+                    {
+                new AllowableDifference("<content xmlns=\"http://www.w3.org/2005/Atom\" />","<content type=\"text\" xmlns=\"http://www.w3.org/2005/Atom\" />"),
```

Comment:
The indents seem incorrect. Can you please run CodeFormatter on the test project? It should fix the issues with coding style.

Reply:
Updated.